### PR TITLE
Create firebase and firestore security rules

### PR DESCRIPTION
**Pull request by DrWala**

### CODE REVIEW

File: backend/firebase.rules.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  "rules": {
+    "status": {
+      "$uid": {
+        ".read": "auth != null",
+        ".write": "auth != null && auth.uid == $uid"
+      }
+    }, 
+    "sessions": {
+        "$sess_id": {
+            ".read": "data.child('users').child('0').val() == auth.uid || data.child('users').child('1').val() == auth.uid",
```
Comment (ivantjh) [inline backend/firebase.rules.json:11]: Haha, thats a lot of code to do a simple thing

File: backend/firestore.rules [hunk 2]
```diff
@@ -2,8 +2,7 @@ rules_version = '2';
 service cloud.firestore {
   match /databases/{database}/documents {
     match /{document=**} {
-      allow read, write: if
-          request.time < timestamp.date(2022, 11, 1);
+      allow read, write: if false;
```
Comment (ivantjh) [inline backend/firestore.rules:5]: Hmm, is there something missing here? This looks like it would block Firestore access to everyone.

**Review by ivantjh — CHANGES_REQUESTED** [review]

File: backend/firestore.rules (hunk 2 above)
Comment (DrWala) [reply backend/firestore.rules:5]: No, by default the admin sdk has full access to firestore. Our client never touches firestore, so we can block all public access to it.

File: backend/firebase.rules.json (hunk 1 above)
Comment (DrWala) [reply backend/firebase.rules.json:11]: 🤷‍♂️  It enforces the rules

File: backend/firestore.rules (hunk 2 above)
Comment (ivantjh) [reply backend/firestore.rules:5]: Ahh, ok that sounds right

**Review by ivantjh — APPROVED** [review]
lgtm!

**Pull request merged**